dapp/client/App.elm

```diff
@@ -35,6 +38,9 @@ port requestAddress : () -> Cmd message
 port requestWallet : Address -> Cmd message
 
 
+port requestTokenList : () -> Cmd message
```

Comment:
I suspect there are a number of missing ports on the ELM side, I only just started on the elm side and then went and filled in the JS side.